test/ITensorChainRules/runtests.jl

```diff
@@ -4,6 +4,7 @@ using FiniteDifferences
 using ITensors
 using ITensors.NDTensors
 using ITensorNetworkAD
+using OptimKit
```

Comment:
Can separate this file into two, one testing chainrules another testing  energy minimization?

Reply:
Sure, makes sense.

Reply:
I reorganized the tests in the latest commits.